_pages/en/strategy-open-first.md

```diff
@@ -0,0 +1,445 @@
+
+> Promote competition in digital markets, for all enterprises regardless of size, including in government procurement, encourage and support a dynamic start-up/scale-up culture in the digital and technology sectors, and promote sustainable economic growth through open markets.
+
+This is why the strategy aims to make it possible for ESDC to share software using a process comparable to that for its documents.
+
+Note:
+
+Work done in the open supports greater transparency and accountability, ensuring that citizens can trust the public service in doing due diligence.
+
+It helps contribute to the greater good by sharing knowledge and to ensure durability of investments done with our citizens money.
+
+### Build a world class public service
+
+Open standards and OSS are about co-creation and building public services that connect seamlessly across organisational silos and borders.
+That is why it is a component in most of the actions in the ISA2 programme.
+Building a responsible open Canada is something we can only do ourselves and the best way is to do it together.
+
+A [Study on OSS governance at the European Commission](https://op.europa.eu/en/publication-detail/-/publication/1aa67278-5464-11ea-aece-01aa75ed71a1) shows that their OSS strategy is in line with the best practices from public services and private companies.
+It confirms that the strengthened policy on the use of OSS is the norm nearly everywhere.
+The study's high-level recommendations are:
+
+- emphasise the use and benefits of open source;
+- create a dedicated open source entity that fosters and measures strategy adoption;
+- improve procurement and product-management processes;
+- establish an open culture;
+- collaborate with communities and the OSS ecosystem;
+- manage legal/licence/intellectual property rights issues; and
+- enhance and develop the technical infrastructure and services.
+
+This strategy is also in line with the [European Commission OSS strategy 2020-2023](https://ec.europa.eu/info/sites/default/files/en_ec_open_source_strategy_2020-2023.pdf) while building on the GC Policies, Directives and the Digital Standards.
+
+The rest of the world has realized the value of working in the open by default.
+Canada has a policy in line with the world's best approaches.
+
+By adopting this strategy, ESDC is taking a stand and moves towards a world class public service.
+
+## 5. Governing Principles
+
+This strategy supports the Canada's aims of radical transformation by simplifying and streamlining processes.
+It will help to make collaborative working methods the de-facto standard for the ESDC's work internally and with others.
+
+It is important to stress that the principles and the action plan relate to existing work processes.
+The use of OSS is commonplace across the GC and in many parts of ESDC there is an established OSS-like culture.
```

Comment:
ummmmm....I wish this was true but I'm not sure that it is.  Perhaps, 'we want to move towards a GC where the use of OSS is commonplace and have an established open source-like culture at ESDC. '?  I noticed that the EU doc has  the wording 'open source-like culture'. This makes more sense to me than an OSS-like culture.